src/manifest/extensions/data/ProspectEventsDataExtension.ts

```diff
@@ -0,0 +1,92 @@
+import { UnknownContextKeys } from '../../../context/keys/UnknownContextKeys';
+import { utils } from '../../../utils';
+import { DataExtension } from './DataExtension';
+import { DataExtensionType } from './DataExtensionType';
+import { ProspectEventsHost } from './ProspectEventsHost';
+
+export class ProspectEventsDataExtension extends DataExtension {
+  /**
+   * Type of mailing link data extension
+   * @see https://github.com/getoutreach/extensibility-sdk/blob/master/docs/manifest.md#type
+   * @type {DataExtensionType}
+   * @memberof ProspectEventsDataExtension
+   */
+  public type: DataExtensionType.PROSPECT_EVENTS = DataExtensionType.PROSPECT_EVENTS;
+
+  /**
+   * Definition of hosting section
+   *
+   * @type {{ url: string }}
+   * @memberof ProspectEventsDataExtension
+   */
+  public host: ProspectEventsHost;
+
+  /**
+   * Validates the data extension configuration
+   *
+   * @param {Application} application
+   * @return {*}  {string[]}
+   * @memberof ProspectEventsDataExtension
+   */
+  validate(): string[] {
+    const issues: string[] = [];
+
+    if (!this.host) {
+      issues.push('Host section is missing.');
+    } else {
+      if (this.host.url) {
+        if (!utils.hostUrlValidation(this.host.url, this.context)) {
+          issues.push('Host url is invalid. Value: ' + this.host.url);
+        }
+      }
+
+      if (!this.type || !Object.values(DataExtensionType).includes(this.type as DataExtensionType)) {
+        issues.push('Host type is invalid. Value: ' + this.type);
+      }
+
+      if (!this.host.category) {
+        issues.push('Host category definition is missing');
```

Comment:
All of those strings here are hardcoded, are we thinking about localisation in this repo?

Reply:
https://outreach-io.atlassian.net/browse/OREE-927 Created for SDK string localization